org.omg.sysml/src/org/omg/sysml/delegate/ConjugatedPortTyping_portDefinition_SettingDelegate.java

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
  * SysML 2 Pilot Implementation
- * Copyright (c) 2022 Siemens AG
```

Comment:
This has been removed because of the change in the filename, but other similar files (which have been similarly thoroughly rewritten, but not renamed) kept the original copyright as well. Probably not an issue, just noticed.

Reply:
This is really an entirely new file. Everything changed from the old `conjugatePortDefinition` file except the boilerplate. I probably should have deleted the old file and created a new one, rather than copying the old one, so that it wouldn't show up in the dif.